src/main/java/edu/ucsb/cs156/happiercows/entities/CowHerd.java

```diff
@@ -0,0 +1,37 @@
+package edu.ucsb.cs156.happiercows.entities;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.Builder;
+import lombok.AccessLevel;
+
+import javax.persistence.*;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+
+/**
+ * Represents the set of cows owned by a user that have the same health.
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Builder
+@Entity(name = "cow_herd")
+public class CowHerd {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @EqualsAndHashCode.Exclude
+    @JsonBackReference
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_commons_id")
+    private UserCommons userCommons;
+
+    private int numCows;
+
+    @Builder.Default
+    private int health = 100;
```

Comment:
Are you referring to more granular binning or a completely different partitioning strategy as I have outlined? This is a trivial fix for the scope of this PR either way, but I am curious to hear how prior groups have found a granularity of 1 is too big whereas a granularity of (I think you mentioned 0.5 previously?) is fine since this is almost a 1:1 design problem I have worked on for a large project.